velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -87,23 +90,150 @@ TEST_F(MapTest, Wide) {
       mapVector);
 }
 
-TEST_F(MapTest, ErrorCases) {
+TEST_F(MapTest, errorCases) {
+  auto inputVectorInt64 = makeNullableFlatVector<int64_t>({1, 2, 3});
```

Comment:
since values do not matter for this tests, we can have fewer vectors

```
auto bigintArg = ...
auto doubleArg = ...
auto nullBigintArg = ...
```

Reply:
Sure, thanks.